.eslintrc.json

```diff
@@ -16,6 +16,8 @@
   "env": {
     "node": true,
     "es2023": true,
+    "browser": true,
     "jest/globals": true
-  }
+  },
+  "ignorePatterns": ["src/public/orderNotifications.js"]
```

Comment:
this `ignorePatterns` line can be removed

Reply:
<img width="725" alt="2" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/73a3019a-df79-4e29-8644-dd7891d77161">